code/processes/dqe.q

```diff
@@ -16,25 +16,80 @@ readdqeconfig:{[file]
 
 gethandles:{exec procname,proctype,w from .servers.SERVERS where (procname in x) | (proctype in x)};
 
-tableexists:{x in tables[]};                                                                                    /- function to check for table, param is table name as a symbol
+tableexists:{[tab]                                                                                              /- function to check for table, param is table name as a symbol
+  result:();
+  .lg.o[`dqe;"checking if ", (string tab), " table exists"];
+  $[1=a:tab in tables[];result:(1b;((string tab)," table exists"));result:(0b;(string tab)," missing from process")]
+  };
 
-runcheck:{[fn;vars;rs]                                                                                          /- function used to send other function to test processes
+tableticking:{[tab;timeperiod;timetype]                                                                         /- function to check if table is increasing in size
+  $[0<a:count select from tab where time within (.z.p-timetype$"J"$string timeperiod;.z.p);                     /- params are tab        - table to check
+    (1b;"there are ",(string a)," records");                                                                    /-            timeperiod - time back to check for records
+    (0b;"the table is not ticking")]                                                                            /-            timetype   - either `minute or `second
```

Comment:
seems a little awkward to me that args to function are explained over subsequent lines of function i.e. the comment on a given line has no relevance to that line?